pkg/controller/managedresources/controller.go

```diff
@@ -115,6 +117,51 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return r.reconcile(mr, log)
 }
 
+func (r *Reconciler) determineClusterIdentity(c client.Client, force bool) (string, error) {
+	cm := corev1.ConfigMap{}
+	err := c.Get(r.ctx, client.ObjectKey{Name: "cluster-identity", Namespace: "kube-system"}, &cm)
+	if err == nil {
+		if id, ok := cm.Data["cluster-identity"]; ok {
+			return id, nil
+		}
+		if force {
+			return "", fmt.Errorf("cannot determine cluster identity from configmap: no cluster-identity entry ")
+		}
+	} else {
+		if force {
+			return "", fmt.Errorf("cannot determine cluster identity from configmap: %s", err)
+		}
+	}
+	return "", nil
+}
+
+func (r *Reconciler) origin(mr *resourcesv1alpha1.ManagedResource) string {
+	var err error
+	if r.cluster == "<cluster>" || r.cluster == "<default>" {
```

Comment:
What is this `<cluster>` and `<default>`? Probably it makes sense to define constants with doc strings for them?

Reply:
yepp, I added constants and documentation for their usage